tests/e2e/specs/mutation.js

```diff
@@ -28,11 +28,14 @@ describe('Mutations component', () => {
       .click({ force: true })
     cy
       .get('.c-mutation-menu-list')
+      .should('be.visible')
       .find('.v-list-item__action')
+      .first()
       .click()
   }
   const submitMutationForms = () => {
     cy.visit('/#/workflows/one')
+    cy.get('.treeitem').should('be.visible')
```

Comment:
Brittle test that started failing as the assertions were executed before the page was fully loaded.